src/mode/terminal.rs

```diff
@@ -278,6 +279,11 @@ where
         self.properties.display_on(on).terminal_err()
     }
 
+    /// Change the display brightness.
+    pub fn change_brightness(&mut self, brightness: Brightness) -> Result<(), TerminalModeError> {
```

Comment:
The other setters are prefixed with `set_`, not `changed_` so for consistency:

```suggestion
    pub fn set_brightness(&mut self, brightness: Brightness) -> Result<(), TerminalModeError> {
```

Reply:
I'm usually against the get/set wording, as explained by Kevlin Henney, those are the most overloaded words in the English language. I know it's against the tradition, but that's the reason if you ever catch me not naming something `set` :)

Reply:
Agreed. I wrote these methods before I knew about the [Rust API guidelines](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter). There's nothing in there about setters, but we could certainly remove the `get_` prefixes in a future PR to bring this crate in line with the guidelines.